lib/features/thread/presentation/thread_controller.dart

```diff
@@ -101,11 +104,11 @@ class ThreadController extends BaseController with EmailActionController {
   bool canSearchMore = false;
   MailboxId? _currentMemoryMailboxId;
   final ScrollController listEmailController = ScrollController();
-  final FocusNode focusNodeKeyBoard = FocusNode();
   final latestEmailSelectedOrUnselected = Rxn<PresentationEmail>();
   @visibleForTesting
   bool isListEmailScrollViewJumping = false;
   WebSocketQueueHandler? _webSocketQueueHandler;
+  FocusNode? focusNodeKeyBoard;
```

Comment:
why we need this?

Reply:
This variable was already initialized (used to listen for the `Shift` key event to select email). I just switched it to `optional` and changed its position.